package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "outline-client",
   "productName": "Outline",
-  "version": "1.2.21",
+  "version": "1.2.24",
```

Comment:
I'll have to think about how to track the version number now that both Windows and Linux are taking their version number from this. Let's just revert for now.